code/modules/religion/festival/instrument_rites.dm

```diff
@@ -0,0 +1,156 @@
+
+/datum/religion_rites/song_tuner/sooth/finish_effect(atom/song_player, datum/song/song_datum)
+	for(var/mob/living/carbon/human/listener in song_datum.hearing_mobs)
+		if(!listener.all_wounds.len)
+			continue
+		var/datum/wound/soothed_wound = pick(listener.all_wounds)
+		soothed_wound.remove_wound()
+
+/datum/religion_rites/song_tuner/pain
+	name = "Murderous Chord"
+	desc = "Sing a sharp song, cutting those around you. Works less effectively on fellow priests. At the end of the song, you'll open the wounds of all listeners."
+	particles_path = /particles/musical_notes/harm
+	song_invocation_message = "You've prepared a painful song!"
+	song_start_message = span_danger("This music cuts like a knife!")
+	glow_color = "#FF4460"
+	repeats_okay = FALSE
+
+/datum/religion_rites/song_tuner/pain/song_effect(atom/song_player, datum/song/song_datum)
+	if(!song_datum)
+		return
+	for(var/mob/living/listener in song_datum.hearing_mobs)
+		if(listener.anti_magic_check(magic = FALSE, holy = TRUE))
+			continue
+		var/pain_juice = 1
+		if(listener.mind?.holy_role)
+			pain_juice *= 0.5
+		listener.adjustBruteLoss(pain_juice)
+
+/datum/religion_rites/song_tuner/pain/finish_effect(atom/song_player, datum/song/song_datum)
+	for(var/mob/living/carbon/human/listener in song_datum.hearing_mobs)
+		var/obj/item/bodypart/sliced_limb = pick(listener.bodyparts)
+		sliced_limb.force_wound_upwards(/datum/wound/slash/moderate/many_cuts)
+
+/datum/religion_rites/song_tuner/lullaby
+	name = "Spiritual Lullaby"
+	desc = "Sing a lullaby, tiring those around you, making them slower. At the end of the song, you'll put people who are tired enough to sleep."
+	particles_path = /particles/musical_notes/sleepy
+	song_invocation_message = "You've prepared a sleepy song!"
+	song_start_message = span_warning("This music's making you feel drowsy...")
+	favor_cost = 40 //actually really strong
+	glow_color = "#83F6FF"
+	repeats_okay = FALSE
+	///assoc list of weakrefs to who heard the song and
+	var/list/listener_counter = list()
+
+/datum/religion_rites/song_tuner/lullaby/Destroy()
+	QDEL_NULL(listener_counter)
+	. = ..()
+
+/datum/religion_rites/song_tuner/lullaby/song_effect(atom/song_player, datum/song/song_datum)
+	if(!song_datum)
+		return
+	for(var/mob/living/listener in song_datum.hearing_mobs)
+		if(listener.anti_magic_check(magic = FALSE, holy = TRUE))
+			continue
+		if(listener.mind?.holy_role)
+			continue
+		if(prob(20))
+			to_chat(listener, span_warning(pick("The music is putting you to sleep...", "The music makes you nod off for a moment.", "You try to focus on staying awake through the song.")))
+			listener.emote("yawn")
+		listener.blur_eyes(2)
+
+/datum/religion_rites/song_tuner/lullaby/finish_effect(atom/song_player, datum/song/song_datum)
+	for(var/mob/living/carbon/human/listener in song_datum.hearing_mobs)
+		to_chat(listener, span_danger("Wow, the ending of that song was... pretty..."))
+		listener.AdjustSleeping(5 SECONDS)
```

Comment:
vine boom sound effect

Reply:
should probably add an ear protection check to this just to make sure that you can't get a free gank on a seccie from it by cooking the song off screen for ~20 seconds

Reply:
Is hearing is part of hearingmobs, will add antimagic